tests/wpt/metadata/MANIFEST.json

```diff
@@ -37215,6 +37215,12 @@
     "deleted_reftests": {},
     "items": {
       "testharness": {
+        "dom/nodes/NodeList-Iterable.html": [
+          {
+            "path": "dom/nodes/NodeList-Iterable.html",
+            "url": "/dom/nodes/NodeList-Iterable.html"
+          }
+        ],
```

Comment:
Never mind :) it passed the CI so it is good. I am just surprised that the new entry it the first one in `items`